pkg/volume/csi/csi_plugin.go

```diff
@@ -60,6 +64,16 @@ var _ volume.VolumePlugin = &csiPlugin{}
 func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	glog.Info(log("plugin initializing..."))
 	p.host = host
+
+	// Initalizing csiDrivers map and label management channels
+	csiDrivers = sync.Map{}
```

Comment:
Ideally this should also be a class instead of a raw map, but we can do that refactor in the future.

Reply:
will do it in a followup PR